### ISSUE TITLE
After Successful Login Automatically Show the App

### ISSUE THREAD

**Issue by nelsonic** [dwyl/app-mvp-elm#43]
At present, after successful authentication, we are presented with this screen:
![image](https://user-images.githubusercontent.com/194400/78540902-b91e9d80-77ec-11ea-9845-6d4f6fb7b97f.png)

As a user I don't want to have to click a button/link before seeing the app interface.

# Todo

+ [x] Automatically show the interface with my items/timers
+ [x] Automatically "focus" the cursor on the text area to input an item.

**Comment by SimonLab** [comment]
I agree with displaying the list of captures once the user is authenticated.
This page can even be removed entirely as we can display the navigations at the top of the pages.

**Comment by SimonLab** [comment]
Removing the current home page means redefining the routes and url parser:
https://github.com/dwyl/app-mvp-elm/blob/e7edc1a728c71512e850f2acdf302a5ba0a0fd73/src/Route.elm#L15-L31

We also need to make sure to redirect to the authentication page when ther user is not logged in anymore.

**Comment by SimonLab** [comment]
Creating a `login` page where the user is redirected when not authenticated yet.
Currently the `not logged in yet` message needs to be created on all the pages:
https://github.com/dwyl/app-mvp-elm/blob/0fe2775c41d5a1ee4c16fb2d781127876daab721/src/Pages/Capture.elm#L224-L230

We can instead check the session of the user when the page is initialised and redirect to the new login page.

**Comment by nelsonic** [comment]
@SimonLab the login page will be covered by Auth. 👍

**Comment by SimonLab** [comment]
Yes at the moment the auth page is currently displayed when the session of the user is "guest":
![image](https://user-images.githubusercontent.com/6057298/78574440-eb96bd80-7821-11ea-9d31-57a388fc996d.png)

I wanted to see if a login page could provide a nicer transition as for now when a session expired or when the user logout, the application redirect automatically to this auth page. This felt a bit abrupt and I think having a login page would help to explain why the user is not logged in anymore (session expired, logout...) and the user can choose to login again.
So the login page will only have some explanation text and a login link which redirect to the auth service. We can try this flow and if this step is not necessary it will be easy to remove.

**Comment by nelsonic** [comment]
@SimonLab what you are describing is a great feature to be added to https://github.com/dwyl/auth 💡 
Let's open a _separate_ issue for that. 👍
